habitat/config/default.py

```diff
@@ -223,28 +223,65 @@ def __init__(self, *args, **kwargs):
 SIMULATOR_SENSOR = CN()
 SIMULATOR_SENSOR.HEIGHT = 480
 SIMULATOR_SENSOR.WIDTH = 640
-SIMULATOR_SENSOR.HFOV = 90  # horizontal field of view in degrees
 SIMULATOR_SENSOR.POSITION = [0, 1.25, 0]
 SIMULATOR_SENSOR.ORIENTATION = [0.0, 0.0, 0.0]  # Euler's angles
+
+# -----------------------------------------------------------------------------
+# CAMERA SENSOR
+# -----------------------------------------------------------------------------
+CAMERA_SIM_SENSOR = SIMULATOR_SENSOR.clone()
+CAMERA_SIM_SENSOR.HFOV = 90  # horizontal field of view in degrees
+CAMERA_SIM_SENSOR.SENSOR_SUBTYPE = "PINHOLE"
+
+SIMULATOR_DEPTH_SENSOR = SIMULATOR_SENSOR.clone()
+SIMULATOR_DEPTH_SENSOR.MIN_DEPTH = 0.0
+SIMULATOR_DEPTH_SENSOR.MAX_DEPTH = 10.0
+SIMULATOR_DEPTH_SENSOR.NORMALIZE_DEPTH = True
+
 # -----------------------------------------------------------------------------
 # RGB SENSOR
 # -----------------------------------------------------------------------------
-_C.SIMULATOR.RGB_SENSOR = SIMULATOR_SENSOR.clone()
+_C.SIMULATOR.RGB_SENSOR = CAMERA_SIM_SENSOR.clone()
 _C.SIMULATOR.RGB_SENSOR.TYPE = "HabitatSimRGBSensor"
 # -----------------------------------------------------------------------------
 # DEPTH SENSOR
 # -----------------------------------------------------------------------------
-_C.SIMULATOR.DEPTH_SENSOR = SIMULATOR_SENSOR.clone()
+_C.SIMULATOR.DEPTH_SENSOR = CAMERA_SIM_SENSOR.clone()
+_C.SIMULATOR.DEPTH_SENSOR.merge_from_other_cfg(SIMULATOR_DEPTH_SENSOR)
 _C.SIMULATOR.DEPTH_SENSOR.TYPE = "HabitatSimDepthSensor"
-_C.SIMULATOR.DEPTH_SENSOR.MIN_DEPTH = 0.0
-_C.SIMULATOR.DEPTH_SENSOR.MAX_DEPTH = 10.0
-_C.SIMULATOR.DEPTH_SENSOR.NORMALIZE_DEPTH = True
 # -----------------------------------------------------------------------------
 # SEMANTIC SENSOR
 # -----------------------------------------------------------------------------
-_C.SIMULATOR.SEMANTIC_SENSOR = SIMULATOR_SENSOR.clone()
+_C.SIMULATOR.SEMANTIC_SENSOR = CAMERA_SIM_SENSOR.clone()
 _C.SIMULATOR.SEMANTIC_SENSOR.TYPE = "HabitatSimSemanticSensor"
 # -----------------------------------------------------------------------------
+# EQUIRECT SENSOR
+# -----------------------------------------------------------------------------
+_C.SIMULATOR.EQUIRECT_RGB_SENSOR = SIMULATOR_SENSOR.clone()
+_C.SIMULATOR.EQUIRECT_RGB_SENSOR.TYPE = "HabitatSimEquirectangularRGBSensor"
+_C.SIMULATOR.EQUIRECT_DEPTH_SENSOR = SIMULATOR_SENSOR.clone()
+_C.SIMULATOR.EQUIRECT_DEPTH_SENSOR.merge_from_other_cfg(SIMULATOR_DEPTH_SENSOR)
+_C.SIMULATOR.EQUIRECT_DEPTH_SENSOR.TYPE = (
+    "HabitatSimEquirectangularDepthSensor"
+)
+# -----------------------------------------------------------------------------
+# FISHEYE SENSOR
+# -----------------------------------------------------------------------------
+FISHEYE_SIM_SENSOR = SIMULATOR_SENSOR.clone()
+FISHEYE_SIM_SENSOR.HEIGHT = FISHEYE_SIM_SENSOR.WIDTH
+FISHEYE_SIM_SENSOR.FOCAL_LENGTH = [FISHEYE_SIM_SENSOR.HEIGHT * 0.5] * 2
```

Comment:
Some comments what 0.5 and 2 magical constants are about. Thank you!

Reply:
TODO: Update params to match habitat_sim